analysis/map.R

```diff
@@ -8,29 +8,58 @@ exposures_by_hasc <- risks_table_mapped %>%
   group_by(hasc) %>%
   summarize(exposures = sum(exposure))
 
+bra_layer0 <- sf::st_read("external_data/gadm36_BRA_gpkg/gadm36_BRA.gpkg", layer = "gadm36_BRA_0")
 bra_layer1 <- sf::st_read("external_data/gadm36_BRA_gpkg/gadm36_BRA.gpkg", layer = "gadm36_BRA_1")
 
+bra_layer0 <- bra_layer0 %>% rmapshaper::ms_simplify()
+bra_layer1 <- bra_layer1 %>% rmapshaper::ms_simplify()
+
 brazil <- bra_layer1 %>%
   left_join(exposures_by_hasc, by = c(HASC_1 = "hasc"))
 
 bins <- c(0, 10^3,10^4, 10^5, 10^6, Inf)
 pal <- colorBin("YlOrRd", domain = brazil$exposures, bins = bins)
 
+bra_bbox <- sf::st_bbox(bra_layer1) %>% unname
+
+earth <-
```

Comment:
Could we move the `sf::st_polygon()` call to be on the same line as `earth`, get rid of `x = `, and put the `list()` on a new line? See https://style.tidyverse.org/syntax.html#long-lines.

Reply:
Actually, let's do:
```
earth <- sf::st_polygon(list(rbind(
  c(360, -360),
  c(360, 360),
  c(-360, 360),
  c(-360, -360),
  c(360, -360)
))) %>%
  sf::st_sfc(crs = sf::st_crs(bra_layer0), check_ring_dir = T) %>%
  sf::st_sf()
```
TIL about https://github.com/r-lib/styler :)